shell/browser/bluetooth/electron_bluetooth_delegate.cc

```diff
@@ -136,9 +162,46 @@ void ElectronBluetoothDelegate::ShowDevicePairPrompt(
     PairPromptCallback callback,
     PairingKind pairing_kind,
     const absl::optional<std::u16string>& pin) {
-  NOTIMPLEMENTED();
-  std::move(callback).Run(BluetoothDelegate::PairPromptResult(
-      BluetoothDelegate::PairPromptStatus::kCancelled));
+  auto* web_contents = content::WebContents::FromRenderFrameHost(frame);
+  if (web_contents) {
```

Comment:
_(nit)_ could simplify the code flow a little by returning here if there's no web_contents

```c++
  auto* web_contents = content::WebContents::FromRenderFrameHost(frame);
  if (!web_contents) {
    return;
  }

  auto* permission_manager = ...
```